zds/utils/templatetags/french_typography.py

```diff
@@ -0,0 +1,21 @@
+from django import template
+from django.template.defaultfilters import stringfilter
+from django.utils.safestring import mark_safe
+
+register = template.Library()
+
+
+@register.filter()
+@stringfilter
+def french_typography(str):
```

Comment:
De la documentation pour ce filtre serait la bienvenue. (Pourquoi tu colories n'importe comment, GitHub ?)

```suggestion
def french_typography(str):
    """
    Replaces spaces with non-breaking-spaces or narrow non-breaking-spaces
    before or after some symbols, according to French typography.
    
    This filter is naive and should not be used on Markdown content.
    """
```